packages/ui/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@orbitkit/ui",
   "version": "0.1.0",
+  "private": true,
```

Comment:
Effectively we could bin this private attribute, and instead include the ui package in the ignore array on our changesets config. 

The [documentation](https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#ignore-array-of-packages) does highlight following however:

> THIS FEATURE IS DESIGNED FOR TEMPORARY USE TO ALLOW CHANGES TO BE MERGED WITHOUT PUBLISHING THEM - If you want to stop a package from being published at all, set `private: true` in its `package.json`.

Reply:
No I think it's fine because then the user of the starterkit just removes private: true from the packages they want to release.